user_docs/en/userGuide.t2t

```diff
@@ -2956,6 +2957,19 @@ Please see the display's documentation for descriptions of where these keys can
 | ToggleHID keyboard input simulation | esytime):l1+joystick1Down, esytime):l8+joystick1Down |
 %kc:endInclude
 
+++ Nattiq nBraille Displays ++[NattiqTechnologies]
+NVDA supports displays from [Nattiq Technologies https://www.nattiq.com/] when connected via USB.
+Windows 10 detects the Braille Displays once connected, you may need to install USB drivers if using older versions of Windows (below Win10). You can get them from the manufacturer website.
+
+Following are the key assignments for Nattiq Technologies displays with NVDA.
+Please see the display's documentation for descriptions of where these keys can be found.
+%kc:beginInclude
+|| Name | Key |
+| Scroll braille display back | up |
+| Scroll braille display forward | down |

```

Comment:
Could you add the previous/next line assignments?

Reply:
ops, sorry, I added them now.

Reply:
@leonardder 
if there is anything else that seems missing, tell me so that I edit it out.
Thanks again.